bin/server.sh

```diff
@@ -44,6 +44,14 @@ check_jmx_status;
 # save LANG
 set_locale_lang;
 
+if [ $FORKING == "1" ]; then
+    JAVA_RUN="exec ${JAVA_RUN}"
```

Comment:
no and yes. it will be executed after string expansion on line 55 and that will exit the script by replacing it with the Java process. what is the intended behaviour.